content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -184,7 +184,8 @@ For a reference to old feature gates that are removed, please refer to
 | `SELinuxMountReadWriteOncePod` | `false` | Alpha | 1.25 | 1.26 |
 | `SELinuxMountReadWriteOncePod` | `false` | Beta | 1.27 | 1.27 |
 | `SELinuxMountReadWriteOncePod` | `true` | Beta | 1.28 | |
-| `SchedulerQueueingHints` | `true` | Beta | 1.28 | |
+| `SchedulerQueueingHints` | `true` | Beta | 1.28.0 | 1.28.5 |
+| `SchedulerQueueingHints` | `false` | Beta | 1.29 | |
```

Comment:
IIUC, SchedulerQueueingHints will be disabled by default in 1.28.6+ as well later.

Reply:
Let's make not that change until it's true.